redbot/cogs/streams/streams.py

```diff
@@ -760,9 +764,11 @@ async def _get_mention_str(self, guild: discord.Guild) -> Tuple[str, List[discor
         if await settings.mention_here():
             mentions.append("@here")
         can_manage_roles = guild.me.guild_permissions.manage_roles
+        for channel in guild.channels:
+            can_mention_everyone = channel.permissions_for(guild.me).mention_everyone
```

Comment:
As I said before, you need to add `channel` param to `_get_mention_str()` method so that the bot checks the permission for the channel it will send the message in. Iterating through all channels here doesn't make much sense - we only care about permissions in *that specific channel* the message will be sent in.

Reply:
understood. param added, iteration removed